wls-eai-service/src/main/java/de/muenchen/oss/wahllokalsystem/eaiservice/rest/wahlbeteiligung/WahlbeteiligungController.java

```diff
@@ -0,0 +1,21 @@
+package de.muenchen.oss.wahllokalsystem.eaiservice.rest.wahlbeteiligung;
+
+import de.muenchen.oss.wahllokalsystem.eaiservice.rest.wahlbeteiligung.dto.WahlbeteiligungsMeldungDTO;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/wahlbeteiligung")
```

Comment:
Ist der neue Pfad "/wahlbeteiligung" bewusst abweichend vom alten"/saveWahlbeteiligung"?

Reply:
ja, Aktionen sollten entsprechend REST über die Http-Methode gesteuert werden: save -> `put` oder `post`. In der URL sind die Ressourcen referenziert, wie in diesem Fall die Wahlbeteiligung.